### DISCUSSION TITLE
Is anyone interested in headlessUI for Yew/Rust?

### DISCUSSION THREAD

**Discussion by thclark** [tailwindlabs/headlessui#2376] [Ideas]
The amazing [toolchain](https://trunkrs.dev/) and the general development experience in Rust makes [Yew](https://yew.rs/) a great candidate framework for front end development. 

I'm sure [full access to the DOM will happen](https://www.reddit.com/r/rust/comments/s4pw6n/frontend_rust_framework_performance_prognosis/), removing the slight [performance overhead](https://krausest.github.io/js-framework-benchmark/current.html) that presently exists (v0.20 of yew broadly comparable to but a bit slower than react 18).

Tailwind support out of the box in the toolchain is [coming soon](https://github.com/thedodd/trunk/pull/505), although of course is already trivially achievable with a [couple of lines of config](https://dev.to/arctic_hen7/how-to-set-up-tailwind-css-with-yew-and-trunk-il9), anyhow.

One thing that's a struggle is the need to build all the component interactions yourself. **Has anyone thought of / discussed building a headlessUI for Yew (or other rust based frontend framework)?**

**Comment by richardmon** [comment]
I'll love to help, I was building a small app and realized that something like this was severely missing.
